packages/typography/docs/Heading.stories.tsx

```diff
@@ -1,154 +1,119 @@
-          rebrands when our colors change.
-        </li>
-        <li>
-          <strong>Semantic, logical HTML should always take priority</strong>{" "}
-          when deciding which <code>tag</code> to use:
-          <ul>
-            <li>
-              Heading elements should never skip a level, e.g. an{" "}
-              <code>h2</code> followed by an <code>h4</code> should be avoided.
-            </li>
-            <li>
-              The main heading on the page that accurately summarizes the page
-              should be an <code>h1</code>.
-            </li>
-            <li>
-              A <code>Heading</code> should not have a bigger <code>tag</code>{" "}
-              than another visually smaller <code>Heading</code>.
-            </li>
-            <li>
-              <strong>
-                Illogical heading orders will cause pages to fail WCAG
-                compliance tests &amp; audits.
-              </strong>
-            </li>
-          </ul>
-        </li>
-      </ul>
-    </Paragraph>
-  </Box>
+export const Display0 = args => (
+  <Heading {...args}>Have the courage to be vulnerable.</Heading>
 )
-
-export const Display0 = () => <Heading variant="display-0">Display 0</Heading>
-
-export const Heading1 = () => (
-  <Heading data-automation-id="test" variant="heading-1" id="make-me-unique">
-    Heading 1
-  </Heading>
-)
-
-export const Heading2 = () => <Heading variant="heading-2">Heading 2</Heading>
-
-export const Heading3 = () => <Heading variant="heading-3">Heading 3</Heading>
-
-export const Heading4 = () => <Heading variant="heading-4">Heading 4</Heading>
-
-export const Heading5 = () => <Heading variant="heading-5">Heading 5</Heading>
-
-export const Heading6 = () => <Heading variant="heading-6">Heading 6</Heading>
-
-export const Heading1Positive = () => (
-  <Heading variant="heading-1" color="positive">
-    Heading 1 (positive)
-  </Heading>
-)
-
-export const Heading1Negative = () => (
-  <Heading variant="heading-1" color="negative">
-    Heading 1 (negative)
-  </Heading>
-)
-
-export const Heading3Positive = () => (
-  <Heading variant="heading-3" color="positive">
-    Heading 3 (positive)
-  </Heading>
-)
-
-export const Heading3Negative = () => (
-  <Heading variant="heading-3" color="negative">
-    Heading 3 (negative)
-  </Heading>
-)
-
-export const Heading1White = () => (
+Display0.storyName = "Heading"
+Display0.args = { variant: "heading-1", color: "dark" }
+Display0.parameters = {
+  chromatic: { disable: false },
+}
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => (
   <>
-    <Heading variant="heading-1" color="white">
-      Heading 1 (white)
-    </Heading>
-    <Documentation reversed />
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.RowHeader headings={["Default"]} />
+      <StoryWrapper.Row rowTitle="Display 0">
+        <Heading variant="display-0" color={isReversed ? "white" : "dark"}>
```

Comment:
Up to you about this one, but you could make a `const color` to reduce the ternary duplication